.github/config-schema.json

```diff
@@ -1045,6 +1045,29 @@
         }
       ]
     },
+    "mise": {
+      "default": {
+        "detect_extensions": [],
+        "detect_files": [
+          ".mise.toml",
+          ".mise.local.toml"
+        ],
```

Comment:
There are many more possible config file locations; see https://mise.jdx.dev/configuration.html

If there's no particular reason against that, I think all should be supported?

Reply:
Unless something has changed since I wrote this Starship itself is very limited in this regard.

It will not detect files in sub-directories such as `.mise/config.toml`.
And you cannot supply a wildcard pattern like `.mise.*.toml` or `**/.mise.toml`.

So the only thing I could do is guess common environment names like `.mise.prod.toml`, but `detect_files` is user configurable so I'd rather leave that for users to decide.

Reply:
Ah, okay added those.